capstone/src/test/java/com/google/univiz/api/CollegeDataConverterTest.java

```diff
@@ -0,0 +1,73 @@
+package com.google.univiz.api;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.univiz.CarnegieSizeDegree;
+import com.google.univiz.CollegeData;
+import com.google.univiz.CollegeId;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CollegeDataConverterTest {
+
+  private static final int COLLEGE_ID = 193900;
+  private static final String NAME = "New York University";
+  private static final String CITY = "New York";
+  private static final boolean FLAG_MAIN_CAMPUS = true;
+  private static final double LATITUDE = 40.729452;
+  private static final double LONGITUDE = -73.997264;
+  private static final int CARNEGIE_SIZE_DEGREE = 17;
+  private static final double ADMISSION_RATE = 0.1999;
+  private static final double AVG_SAT = 1419.0;
+  private static final int NUM_OF_UNDERGRADS = 26339;
+  private static final int AVG_COST = 69830;
+  private static final double RATIO_OF_MEN = 0.4253;
+  private static final double RATIO_OF_WOMEN = 0.5747;
```

Comment:
Now that these constants are replicated across numerous tests, it might be nice to refactor them into a common spot. You could even factor in a CollegeData test object that would simplify test quite a bit :)

Let's do this in a follow up PR (low priority).